### PR TITLE
New questions and answers

### PR DESCRIPTION
**Pull request by victorwlin**

### CODE REVIEW

File: accounting/accounting-quiz.md [hunk 1]
```diff
@@ -248,3 +248,112 @@
...
+
+#### Q33. Which statement about current liabilities is true?
+
+- [ ] Current liabilities are obligations owed after a 12-month period.
+- [ ] Current liabilities should be included under long-term liabilities.
+- [ ] Current liabilities are obligations owed over 5 years.
+- [x] Current liabilities are obligations due within a year.
+
+> Current liabilities are a company's short-term financial obligations that are due within one year or within a normal operating cycle.  
+> [Reference](https://www.investopedia.com/terms/c/currentliabilities.asp)
```
Comment (Ebazhanov) [inline accounting/accounting-quiz.md:260]:
```suggestion
[Reference](https://www.investopedia.com/terms/c/currentliabilities.asp)
```

File: accounting/accounting-quiz.md [hunk 2]
```diff
@@ -248,3 +248,112 @@
...
+
+#### Q34. How do variable costs and fixed costs act within the relevant range?
+
+- [x] Variable costs per unit remain constant and fixed costs per unit vary.
+- [ ] Fixed costs per unit remain constant and variable costs per unit vary.
+- [ ] Both total variable costs and total fixed costs vary.
+- [ ] Both total variable costs and total fixed costs remain constant.
+
+> While variable costs tend to remain flat, the impact of fixed costs on a company's bottom line can change based on the number of products it produces.  
+> [Reference](https://www.investopedia.com/ask/answers/032515/what-difference-between-variable-cost-and-fixed-cost-economics.asp)
```
Comment (Ebazhanov) [inline accounting/accounting-quiz.md:270]:
```suggestion
[Reference](https://www.investopedia.com/ask/answers/032515/what-difference-between-variable-cost-and-fixed-cost-economics.asp)
```

File: accounting/accounting-quiz.md [hunk 3]
```diff
@@ -248,3 +248,112 @@
...
+#### Q35. Which answer can be defined as an investment center's contribution margin less the fixed costs that are traceable to the investment center?
+
+- [ ] net income
+- [ ] residual income
+- [x] segment margin
+- [ ] return on investment (ROI)
+
+> Contribution Margin = Sales Revenue – Variable Costs
+> Segment Margin = Segment’s Contribution Margin – Fixed Costs traced to the Segment  
+> [Reference](https://qsstudy.com/segment-margin-differs-contribution-margin-explain/)
```
Comment (Ebazhanov) [inline accounting/accounting-quiz.md:281]:
```suggestion
[Reference](https://qsstudy.com/segment-margin-differs-contribution-margin-explain/)
```

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
Hello, @victorwlin! Thanks for PR, can you please improve formatting for links, thanks

**Comment by victorwlin** [comment]
I believe this is the intention, right? To move the reference out of the blockquote?

**Comment by Ebazhanov** [comment]
> I believe this is the intention, right? To move the reference out of the blockquote?

yes with this `[]()` the link become active, you can find more examples in other quizes, thanks for updates

**Review by Ebazhanov — APPROVED** [review]
LTGM, thank you @victorwlin 👍

**Pull request merged**